Homework14(Photos)/Sources/ViewControllers/AlbumViewController.swift

```diff
@@ -0,0 +1,259 @@
+                    bottom: .zero,
+                    trailing: 5)
+
+                let groupSize = NSCollectionLayoutSize(
+                    widthDimension: .fractionalWidth(0.47),
+                    heightDimension: .estimated(470 / 2))
+                let layoutGroup = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [layoutItem])
+
+                let layoutSection = NSCollectionLayoutSection(group: layoutGroup)
+                layoutSection.contentInsets = NSDirectionalEdgeInsets(
+                    top: 5,
+                    leading: 10,
+                    bottom: 20,
+                    trailing: 10)
+                layoutSection.orthogonalScrollingBehavior = .groupPaging
+
+                // Header
+                let layoutSectionHeaderSize = NSCollectionLayoutSize(
+                    widthDimension: .fractionalWidth(1),
+                    heightDimension: .fractionalWidth(0.1))
+                let layoutSectionHeader = NSCollectionLayoutBoundarySupplementaryItem(
+                    layoutSize: layoutSectionHeaderSize,
+                    elementKind: UICollectionView.elementKindSectionHeader,
+                    alignment: .top)
+                layoutSection.boundarySupplementaryItems = [layoutSectionHeader]
+
+                return layoutSection
+            default:
+                let itemSize = NSCollectionLayoutSize(
+                    widthDimension: .fractionalWidth(1),
+                    heightDimension: .fractionalHeight(1))
+                let layoutItem = NSCollectionLayoutItem(layoutSize: itemSize)
+
+                let groupSize = NSCollectionLayoutSize(
+                    widthDimension: .fractionalWidth(1),
+                    heightDimension: .estimated(46))
+                let layoutGroup = NSCollectionLayoutGroup.vertical(layoutSize: groupSize, subitems: [layoutItem])
+
+                let layoutSection = NSCollectionLayoutSection(group: layoutGroup)
+                layoutSection.contentInsets = NSDirectionalEdgeInsets(
+                    top: 10,
+                    leading: 15,
+                    bottom: 20,
+                    trailing: 15)
+
+                // Header
+                let layoutSectionHeaderSize = NSCollectionLayoutSize(
+                    widthDimension: .fractionalWidth(1),
+                    heightDimension: .fractionalWidth(0.1))
+                let layoutSectionHeader = NSCollectionLayoutBoundarySupplementaryItem(
+                    layoutSize: layoutSectionHeaderSize,
+                    elementKind: UICollectionView.elementKindSectionHeader,
+                    alignment: .top)
+                layoutSection.boundarySupplementaryItems = [layoutSectionHeader]
+
+                return layoutSection
+            }
+        }
+    }
+}
+
+extension AlbumViewController: UICollectionViewDataSource {
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        4
+    }
+
```

Comment:
Оперировать цифрами не есть хорошо, для этого создадим перечисление и будем оперировать кейсам!

```suggestion
extension AlbumViewController: UICollectionViewDataSource {
private enum Sections: Int, CaseIterable { // нейминг кейсов по хорошему должен отражать что за секция
        case first = 0
        case second = 1
        case third = 2
        case fourth = 3
    }

    func numberOfSections(in collectionView: UICollectionView) -> Int {
         let sections = Sections.allCases.count
        return sections
    }

```